src/docs/primitives.js

```diff
@@ -89,6 +90,7 @@ export default {
   ],
   resample: ["operator", "Resample data to new dimensions with a shader"],
   repeat: ["operator", "Repeat data in one or more dimensions"],
+  reverse: ["operator", "Reverse data in one or more dimensions"],
```

Comment:
#32 's changelog did not have an entry for reverse...This description is probably true? If vague?

Reply:
Haha I was stymied as well when I looked at this yesterday. If it becomes clear while porting the examples I'll add to these.